src/app/app.module.ts

```diff
@@ -201,6 +201,7 @@ import localeSv from '@angular/common/locales/sv';
 import localeUk from '@angular/common/locales/uk';
 import localeZhCn from '@angular/common/locales/zh-Hans';
 import localeZhTw from '@angular/common/locales/zh-Hant';
+import { BulkActionsComponent } from './vault/bulk-actions.component';
```

Comment:
should be sorted with other component imports

Reply:
Done